drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/TestToolsPresenter.java

```diff
@@ -197,18 +202,11 @@ public void setEventBus(EventBus eventBus) {
         this.eventBus = eventBus;
     }
 
-    @Override
-    public void setGridWidget(GridWidget gridWidget) {
+    protected void setGridWidget(GridWidget gridWidget) {
         this.gridWidget = gridWidget;
-        switch (gridWidget) {
-            case BACKGROUND:
-                hideInstances();
-                break;
-            case SIMULATION:
-                showInstanceListContainerSeparator(true);
-                break;
-            default:
-                throw new IllegalArgumentException("Illegal GridWidget " + gridWidget);
+        onDisableEditorTab();
+        if (GridWidget.BACKGROUND.equals(gridWidget)) {
```

Comment:
@dupliaka 
Hi Anna. This is an long-lasting topic, anyway from a technical point of view there is absolutely no reason to use "Object.equals" when one of the terms is surely not null - the `GridWidget.BACKGROUND` enum, in this case. I personally prefer to avoid `Object.equals` in such situations because it is confusing - anyway it is just my personal opinion. (By side note: in general I always prefer to avoid unneeded code only for "uniformity" - again just my preference)

Reply:
Reasonable. Then we shell state it as it is.